Source/RunActivity/Viewer3D/MSTSSky.cs

```diff
@@ -68,11 +68,12 @@ public class MSTSSkyDrawer
 
         public Orts.Viewer3D.SkyViewer.Date date;
 
+        private SkySteps skySteps = new SkySteps();
         // Size of the sun- and moon-position lookup table arrays.
         // Must be an integral divisor of 1440 (which is the number of minutes in a day).
-        private int maxSteps = 72;
-        private double mstsskyoldClockTime;
-        private int step1, step2;
+        //private int maxSteps = 72;
+        //private double mstsskyoldClockTime;
+        //private int step1, step2;
```

Comment:
There are commented out lines here too that could be deleted.

Reply:
Thanks, Peter.

I didn't mean to leave in this old code. Thanks for spotting it.

Guess I'm trying to do too much at once !

Chris